exercises/Anne-Hyacinthe/answers.js

```diff
@@ -0,0 +1,130 @@
+// Question 1
+function greet(name) {
+ return (`Good morning, ${name}!`);
+}
+
+// Question 2
+function logOddNumbers() {
+  for(i = 1; i <= 99; i += 1)
+    if (i % 2 !== 0){
+      console.log(i);
+    }
+}
+
+// Question 3
+function area(length, width) {
+ return length * width;
+  
+}
+
+// Question 4
+function otherAngle(angle1, angle2) {
+  let MissingAngle = 180 - (angle1 + angle2)
+  return MissingAngle
+}
+
+// Question 5
+function diameterFromArea(area) {
+   let radius2;
+  radius2 = area / Math.PI ;
+  let x;
+  x= Math.sqrt(radius2)
+  let diameter = x * 2 ;
+  return diameter;
+}
+
+// Question 6
+function totalPrice(cost, tax) {
+  percent = cost / 100;
+  per = tax * percent;
+  total = per + cost;
+  return(total);
+}
+
+// Question 7
+function shortLongShort(e,y) {
+   if (e.length > y.length) {
+    return (y+e+y)
+  } else if (y.length > e.length){
+    return (e+y+e)
+  }
+  
+  
+}
+
+// Question 8
+function describeAge(age) {
+  if (age <= 12) {
+    return "You're a kid";
+  }else if (13 <= age && age <= 17){
+    return "You're a teenager";
+  } else if (18 <= age && age <= 64){
+    return "You're an adult";
+  } else {
+    return "You're an eldely";
+  }
+
+
+}
+
+// Question 9
+function isLeapYear(year) {
+if (year === 0){
```

Comment:
Way too many conditions here, Anne. We want to check for two things?
1. Is it divisible by four and not divisible by 100?
OR
2. Is it divisible by 400?

Refactor this solution.